wayland_1_9: remove

globin opened the pull request:
Ancient compat version, that no longer has any use
It was introduced > 3 yrs ago in 0593ad2b1613e22efa88307ee6ee4115563721fb

hedning commented on nixos/modules/tasks/network-interfaces-systemd.nix:
```diff
@@ -72,15 +72,6 @@ in
-          # - https://github.com/NixOS/nixpkgs/issues/18962
-          # - https://github.com/NixOS/nixpkgs/issues/61629
-          matchConfig = mkDefault { Name = "*"; };
-        };
```
This doesn't look related?

globin replied:
Fixed, thanks

globin committed:
wayland_1_9: remove

ancient compat version, that no longer has any use

was introduced > 3 yrs ago in https://github.com/NixOS/nixpkgs/commit/0593ad2b1613e22efa88307ee6ee4115563721fb
